app/src/main/java/org/oppia/app/administratorcontrols/appversion/AppVersionViewModel.kt

```diff
@@ -23,12 +24,11 @@ class AppVersionViewModel @Inject constructor(
 
   // TODO(#555): Create one central utility file from where we should access date format or even convert date timestamp to string from that file.
   private fun getDateTime(l: Long): String? {
```

Comment:
Update:
```
private fun getDateTime(lastUpdateTime: Long): String? {
    return oppiaDateTimeFormatter.formatDateFromDateString(
      oppiaDateTimeFormatter.dd_MMMM_yyyy,
      lastUpdateTime,
      Locale.US
    )
  }
```

Reply:
Done, will remove this todo in the last commit when squash all commits.